api/lib/infrastructure/serializers/jsonapi/organization-invitation-serializer.js

```diff
@@ -4,7 +4,15 @@ module.exports = {
 
   serialize(invitations) {
     return new Serializer('organization-invitations', {
-      attributes: ['organizationId', 'organizationName', 'email', 'status', 'updatedAt'],
+      transform(record) {
+        record.organization = { id: record.organizationId };
+        return record;
+      },
+      attributes: ['organizationId', 'organizationName', 'email', 'status', 'updatedAt', 'organization'],
```

Comment:
L'attribut `organizationId` est redondant.

Au passage l'attribut `organizationName` n'est pas systématiquement valorisé (il est renseigné par `/api/organization-invitations/{id}` mais pas par `/api/organizations/{id}/invitations`), alors que les deux réponses déclarent le même `type` JSON-API.

Reply:
C'est noté, on va voir la différence entre les deux.

Reply:
idem que [au-dessus](https://github.com/1024pix/pix/pull/1858#discussion_r485438863)
=> le fichier n'est plus modifié par le ticket